docs/manufacturing.md

```diff
@@ -1,13 +1,67 @@
 # Babel-Babel manufacturing
 
+## PCB specification for all PCB
+### Main PCB
+* Quantity per product/order: 1
+* Layers: 4 (layer stack in manufacturing package)
+* Thickness: 1.0mm
+* PCB color: Red
+* Silkscreen: White
+* Surface Finish: LeadFree HASL-RoHS
+* Outer Copper Weight: 1 oz or 1.4 Mils
+* Inner Copper Weight: 0.5 oz or 0.7 mils
+* Material Type: FR4-Standard Tg 130-140C
+* Flying Prob Test: Fully Test
+* Castellated Holes: No
+
+### Enclosure PCB
+
+### Part B1
+   * Quantity per product/order: 1
```

Comment:
Please fix the formatting here: add newlines after headings and remove indentation.